### PR TITLE
https://github.com/directus/sdk-js/issues/154

### PR DESCRIPTION
**Pull request by MathiRa**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] API docs have been updated (by executing `npm run documentation`)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

## What is the current behavior?
eg. Issue Number: [154](https://github.com/directus/sdk-js/issues/154)

## What is the new behavior?
Able get all settings independent from default limit because if you use a default limit less than rows of settings is not working.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/SDK.ts [hunk 1]
```diff
@@ -769,8 +769,10 @@ export class SDK {
   /**
    * Get Directus' global settings
    * @param {QueryParamsType?} params
+   * Limit is hardcode in -1 because is necessary get all settings rows.
```
Comment (rijkvanzanten) [inline src/SDK.ts]:
```suggestion
   * Limit is hardcoded to -1 because we always want to return all settings
```

Comment (MathiRa) [reply src/SDK.ts]: Thanks for your correction and I'm sorry for my English I'm working to improve it.

Comment (rijkvanzanten) [reply src/SDK.ts]: Ohh no worries I really appreciate the PR 🙂 

English isn't my native language either, so I know what it's like

Comment (MathiRa) [reply src/SDK.ts]: Really thanks a lot!

**Pull request merged**